optimus/optimus.py

```diff
@@ -171,14 +171,14 @@ def __init__(self, session=None, master="local[*]", app_name="optimus", checkpoi
 
     @staticmethod
     def connect(driver=None, host=None, database=None, user=None, password=None, port=None, schema="public",
-                oracle_tns=None, oracle_service_name=None, oracle_sid=None, presto_catalog=None,
-                cassandra_keyspace=None, cassandra_table=None):
+                oracle_service_name=None, oracle_sid=None, presto_catalog=None, cassandra_keyspace=None,
```

Comment:
What happened to the oracle_tns param?

Reply:
None of the connection strings are currently using it:

None of the connection strings use ```oracle_tns```, they use either ```oracle_service_name or ```oracle_sid``` (see oracle.py line 17)

```
return f"""jdbc:{conn_properties["driver"]}:thin:@{conn_properties["host"]}:{conn_properties["port"]}/{conn_properties["oracle_sid"]}""" \
            if conn_properties["oracle_sid"] is not None \
            else f"""jdbc:{conn_properties["driver"]}:thin:@//{conn_properties["host"]}:{conn_properties["port"]}/{conn_properties["oracle_service_name"]}"""
```
I have successfully test oracle using both connectuion strings and they work :)

Reply:
@eschizoid I created https://hub.docker.com/repository/docker/ironmussa/optimus, Do you have a Docker ID so I can add you to the team?

Reply:
> @eschizoid I created https://hub.docker.com/repository/docker/ironmussa/optimus, Do you have a Docker ID so I can add you to the team?

eschizoid

Reply:
Added!